meshtastic/tests/test_node_runtime_transport.py

```
@@ -0,0 +1,1294 @@
+            ch = channel_pb2.Channel()
+            ch.index = i
+            if i == 0:
+                ch.role = channel_pb2.Channel.Role.PRIMARY
+            else:
+                ch.role = channel_pb2.Channel.Role.DISABLED
+            channels.append(ch)
+
+        mock_local_node.channels = channels
+
+        with pytest.raises(_SentinelError, match="interface error"):
+            with mock_local_node._channels_lock:
```

Ruff (0.15.6)

[warning] Use a single `with` statement with multiple contexts instead of nested `with` statements

(SIM117)